ref/general/LDAP-user-registries-authentication-authorization.adoc

```diff
@@ -0,0 +1,56 @@
+
+LDAP, the industry standard for directory servers, provides a means to create, modify, search, and delete users and groups in your applications.
+There are many LDAP servers available, such as the link:https://www.ibm.com/support/knowledgecenter/SSVJJU_6.4.0/com.ibm.IBMDS.doc_6.4/c_po_SDS_overview.html[IBM Security Directory Server] and the Microsoft Active Directory.
+There are also open source LDAP servers, such as OpenLDAP and ApacheDS.
+
+== Federated LDAP registries
+
+You can combine multiple LDAP registries into a single federated registry to manage authentication and authorization.
+Federated registries provide a single logical view of multiple user registries and repositories.
+
+In cases where user and group information is spread across multiple registries, federation enables Open Liberty to use distributed user information in a unified manner, with a continuous store of information.
+Different types of registries can be federated to combine information from LDAP, basic, and custom user registries and repositories into a common registry. However, user registries that are configured with the `quickStartSecurity` element cannot be federated with other registries.
+
+LDAP registries are federated by default.
+If you configure more than one LDAP registry in your `server.xml` file, then the registries are automatically federated into a common registry.
+This default federation occurs regardless of whether the LDAP registries are separate data stores or two subtrees of the same LDAP registry.
+Similarly, if one or more LDAP registries are configured with one or more basic or custom registries in the `server.xml` file, then they are federated by default.
+
+You can adjust the configuration of federated registries by enabling the link:/docs/ref/feature/#federatedRegistry-1.0.html[Federated User Registry] feature and configuring the `federatedRepository` element.
+For more information, see link:/docs/ref/config/#federatedRepository.html[User Registry Federation].
+
+== LDAP registries and the Social Media Login feature
+
+You can use Open Liberty's Social Media Login feature to help manage authentication with a configured LDAP registry.
+Depending how you configure the feature, users can either supply their social media credentials to authenticate or choose between their LDAP login and their social media credentials.
+For more information, see the example configurations for the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login feature].
+
+== LDAP registry configuration
+
+To use an existing LDAP server as a user registry, first add the link:/docs/ref/feature/#appSecurity-3.0.html[Application Security] and link:/docs/ref/feature/#ldapRegistry.html[LDAP User Registry] features to your `server.xml` file.
```

Comment:
"To use an existing LDAP server as a user registry, first add the link:/docs/ref/feature/#appSecurity-3.0.html[Application Security]..."
remove "first" if you can as it sounds like a step and the next sentence is an if statement "If you need to communicate with a TLS-enabled LDAP server," so optional.